resources/zib/zib-BodyWeight.xml

```diff
@@ -0,0 +1,125 @@
+      <definition value="Date and (if possible) time that the weight was measured or estimated." />
+      <alias value="GewichtDatumTijd" />
+      <min value="1" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <type>
+        <code value="Period" />
+      </type>
+      <mapping>
+        <identity value="zib-bodyweight-v3.2-2020EN" />
+        <map value="NL-CM:12.1.4" />
+        <comment value="WeightDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <short value="WeightValue" />
+      <definition value="The patient’s body weight. Body weight is expressed in kilograms (kg). For infants under 3kg, body weight is often expressed in grams. &#xD;&#xA;This concept can also be used to determine an estimated body weight in the event that measuring the exact body weight is not possible - such as for example weighing a resistant child, or in the case of having to estimate the weight of an unborn child." />
+      <alias value="GewichtWaarde" />
+      <min value="1" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <patternQuantity>
+        <system value="http://loinc.org" />
+        <code value="29463-7" />
+      </patternQuantity>
+      <mapping>
+        <identity value="zib-bodyweight-v3.2-2020EN" />
+        <map value="NL-CM:12.1.2" />
+        <comment value="WeightValue" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <short value="Comment" />
+      <definition value="Comment on the weight measurement, such as any problems or factors that may influence body weight at that time, such as timing within the menstrual cycle, timing of recent bowel movement or a note on any amputations." />
+      <alias value="Toelichting" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-bodyweight-v3.2-2020EN" />
+        <map value="NL-CM:12.1.3" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:clothing">
+      <path value="Observation.component" />
+      <sliceName value="clothing" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:clothing.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="248159006" />
+          <display value="State of dress" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:clothing.code.coding">
```

Comment:
I'm not sure if I agree with putting this on `.component`. Components are more about the result than about the measurement. The state of clothing is something that you could imagine to be part of the measurement code (e.g. weighing of fully clothed patient). Therefore, `Observation.method` might be a good element. @ArdonToonstra , @jduwel , what do you think?

Reply:
Agree on component